tests/commands/ironhawk/main_test.go

```diff
@@ -20,13 +21,17 @@ type TestCase struct {
 	name     string
 	commands []string
 	expected []interface{}
+	keysUsed []string
 }
 
 func assertEqual(t *testing.T, expected interface{}, actual *wire.Response) bool {
 	var areEqual bool
 	switch v := expected.(type) {
 	case string:
 		areEqual = v == actual.GetVStr()
+		if strings.HasPrefix(v, "ERR") {
+			areEqual = v == actual.GetErr()
+		}
```

Comment:
We do not want to support `ERR *` strings as errors. that is legacy Redis. We have diverged.